flask_openapi3/utils.py

```diff
@@ -618,4 +618,4 @@ def convert_responses_key_to_string(responses: ResponseDict) -> ResponseStrKeyDi
 
 
 def normalize_name(name: str) -> str:
-    return re.sub(r"[^\w.\-]", "_", name)
+    return re.sub(r"[^\w.\-]", "_", name).rstrip("_")
```

Comment:
Examples:
* `APIResponse_AdminView` instead of `APIResponse_AdminView_`
* `APIResponse_AssayOrder` instead of `APIResponse_AssayOrder_`
* `APIResponse_BlockDeletionReferenceCheckResult` instead of `APIResponse_BlockDeletionReferenceCheckResult_`